client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -46,6 +46,8 @@ function getSupportClassName(
     className = "yes";
     if (version_removed || (flags && flags.length)) {
       className = "no";
+    } else if (getFirst(support) && getFirst(support).hasOwnProperty("notes")) {
```

Comment:
See [this image](https://user-images.githubusercontent.com/159415/156348453-a65414cc-9244-461e-8915-cdb6b3db3fb3.png).

With this check, only the Node.js DateTimeFormat shows an icon. There is an alternative check:

`} else if (asList(support).some(s => s.hasOwnProperty("notes"))) {`

This would show the icon for the first three Node.js rows (everything that previously had a down arrow). For this check to work, I would also need to change the following lines of code:

https://github.com/mdn/yari/blob/main/client/src/document/ingredients/browser-compatibility-table/feature-row.tsx#L224-L226

I would have to change `getSupportClassName(currentSupport, browser)` to `getSupportClassName(support, browser)` in the two places that line appears. This function already does `getFirst(support)` for every other check, so using currentSupport, defined as getFirst(support), is redundant anyway.

What would be nice would be two separate symbols. Before, anything matching the condition I'm currently using had an asterisk, and anything matching this suggested condition had a down arrow. Emulating that behavior in the redesign would be nice.